routes/dms.js

```diff
@@ -161,10 +161,12 @@ module.exports = function () {
   })
 
   router.get('/:owner/:name', async (req, res, next) => {
-    let datapackage = null
+    let datapackage = res.locals.datapackage || null
```

Comment:
Is it documented how res.locals.datapackage should used? I assume that this allows themes / plugins to provide their own datapackage by interceptingthe route via `app.param`

Reply:
@starsinmypockets yes, you also can do it via `app.get('/:owner/:name, ...)` in a theme. I can provide an example: https://gitlab.com/datopian/eds/frontend-eds/blob/master/index.js#L49-61

Reply:
@anuveyatsu This looks fine to merge. Can we document the use of locals here http://tech.datopian.com/frontend

Reply:
@starsinmypockets just to confirm, I'm working on docs for this

Reply:
@starsinmypockets docs are here:
http://tech.datopian.com/frontend/theming/#extending-core-controller